ui/package.json

```diff
@@ -33,6 +33,7 @@
     "dependencies": {
         "azure-arm-resource": "^3.0.0-preview",
         "azure-arm-storage": "^3.1.0",
+        "debug": "^3.1.0",
```

Comment:
So this debug package will now be included with every extension we ship. Are we just saying it's small enough that it doesn't matter?

Reply:
It's pretty small and I do intend to be using it in more places eventually. However, node.js actually has similar functionality built in which I could use instead (uses the NODE_DEBUG variable), although I see debug being used instead in many places, including vscode. I assume the only reason is to avoid the NODE_DEBUG env var.

No strong opinion, I could use the built-in version for now if you'd prefer (https://nodejs.org/api/util.html#util_util_debuglog_section).

Reply:
If VS Code is using it, then I say go ahead and use it

Reply:
Okay, big change in direction.  Thinking about this, I realize I'm using the debug module wrong. Normally if a component is enabled in DEBUG, then it starts getting debug info output to console. If you set DEBUG=*, that would enable it for all modules.  The way I've got it, then would also turn off telemetry and switch to debug telemetry, which is a behavioral change.

I think I should remove debug for now and simply check an environment variable instead, like DEBUGTELEMETRY=1.